tests/test_server/fixtures/factories/lineage.py

```diff
@@ -687,3 +689,62 @@ async def lineage_with_symlinks(
 
     async with async_session_maker() as async_session:
         await clean_db(async_session)
+
+
+@pytest_asyncio.fixture()
+async def branchy_lineage_with_column_lineage(
+    async_session_maker: Callable[[], AbstractAsyncContextManager[AsyncSession]],
+    branchy_lineage: LineageResult,
+) -> AsyncGenerator[LineageResult, None]:
+    # In this fixture we add same column lineage for each operation(O0[D0->D3], O1[D3->D6], O2[D6->D9])
+    # Column lineage will be the same for each operation
```

Comment:
Well, that's very simple case. There could be more, like multiple operations have the same column lineage, or different operations within run have different column lineage. The last one allows testing how type merging is working.

Reply:
BTW, why using complex lineage graph `branchy_lineage` for the simplest column lineage case?